settings.gradle

```diff
@@ -15,3 +15,9 @@ rootProject.name='signers'
 include 'bls-keystore'
 include 'keystorage:hashicorp'
 include 'acceptance-tests'
+include 'signing:secp-signer-api'
```

Comment:
wondering if this should be signing:secp-signer:api (rather than -api)?

Reply:
that's nicer. done

Reply:
gone a bit further and also dropped -signer from the module names. so now it signing:secp:api, signing:secp-azure etc.